src/test/kotlin/DuplicateUnitsTest.kt

```diff
@@ -44,6 +62,14 @@ class DuplicatedUnitsTest {
                     }
                 }
             }
+            val duplicateList =
+                duplicates.flatMap {
+                    it.value.values.flatten().map { typedUnit -> "\"${typedUnit.externalId}\"" }
+                }
+                    .joinToString(",\n", postfix = ",")
+            if (failOnError) {
+                fail("Duplicate units found in the catalog. Update list in EquivalentUnits.kt:\n$duplicateList")
+            }
         } else {
             println("No equivalent units exist in the catalog.")
```

Comment:
Here we could rephrase the print statement based on the new behaviour.
```suggestion
            println("No equivalent units were introduced in this Pull Request.")
```